packages/mobile/src/components/top-tab-bar/CollapsibleTopTabNavigator.tsx

```diff
@@ -91,7 +91,14 @@ export const CollapsibleTabNavigator = ({
       collapsibleOptions={collapsibleOptions}
       initialRouteName={initialScreenName}
       tabBar={tabBar}
-      screenOptions={screenOptions}
+      // lazy={true}
```

Comment:
Not using the lazy option?

Reply:
Sadly no, lazy breaks on collapsible. But the alternative is the "inView" option in lineups to still wait before making request